src/components/table/table.tsx

```diff
@@ -54,31 +59,25 @@ export default function Table(props: TableProps) {
           {headers.length > 0 && (
             <TableHead>
               <TableRow>
-                {headers.map((header, index) => {
-                  return (
-                    <TableCell key={`header-${index}`}>
-                      {capitalizeFirstLetter(header)}
-                    </TableCell>
-                  )
-                })}
+                {headers.map((header, index) => (
+                  <TableCell key={`header-${index}`}>
+                    {capitalizeFirstLetter(header.label || header.dataKey)}
```

Comment:
since users have the option to specify custom labels, should we only capitalize when no labels are provided ?